ocf_data_sampler/load/utils.py

```diff
@@ -27,4 +27,20 @@ def make_spatial_coords_increasing(ds: xr.Dataset, x_coord: str, y_coord: str) -
     assert (ds[x_coord].diff(dim=x_coord) > 0).all()
     assert (ds[y_coord].diff(dim=y_coord) > 0).all()
 
-    return ds
\ No newline at end of file
+    return ds
+
+
+def get_xr_data_array_from_xr_dataset(ds: xr.Dataset, datavar: str | None = None) -> xr.DataArray:
+    """Return underlying xr.DataArray from passed xr.Dataset. If davar provided, returns specified variable. 
+    If datavar not provided, checks only one variable is present and returns it as an xr.DataArray.
+
+    Args:
+        ds: xr.Dataset to extract xr.DataArray from
+        datavar: (if provided) return specific variable from ds
+    """
+    
+    if datavar:
```

Comment:
I think we don't need `datavar` as a parameter. In the places where we are hard coding the variable we should just use `ds.varname` instead of going through this function

Reply:
Good point, I'll reverse this